src/hid_hidapi.c

```diff
@@ -0,0 +1,121 @@
+/* hidapi license text
+Copyright (c) 2010, Alan Ott, Signal 11 Software
```

Comment:
The copyright is normally the author's; if you wrote the code for work, the copyright is usually assigned to your employer (as @djm-google did in hid_openbsd.c).